commit 02b9090edab82021f5e2ecc7f2f4a5fc877949db
Author: Christian Boltz <apparmor@cboltz.de>
Date:   Mon Jul 20 20:35:41 2020 +0200

    abstractions/X: add another xauth path
    
    Future sddm version will use $XDG_RUNTIME_DIR/xauth_XXXXXX
    
    References:
    - https://bugzilla.opensuse.org/show_bug.cgi?id=1174290
    - https://bugzilla.suse.com/show_bug.cgi?id=1174293
    - https://github.com/sddm/sddm/pull/1230
    - https://github.com/jonls/redshift/issues/763
    
    This is the 2.13 version of 35f033ca7c0dbd03111a54ea50b3f2713b9d5584 /
    https://gitlab.com/apparmor/apparmor/-/merge_requests/581
    
    The difference is that this commit avoids using the @{run} variable.

diff --git a/profiles/apparmor.d/abstractions/X b/profiles/apparmor.d/abstractions/X
index 1eca218d..e903861a 100644
--- a/profiles/apparmor.d/abstractions/X
+++ b/profiles/apparmor.d/abstractions/X
@@ -24,6 +24,7 @@
   owner /{,var/}run/lightdm/*/xauthority r,
   owner /{,var/}run/user/*/gdm/Xauthority r,
   owner /{,var/}run/user/*/X11/Xauthority r,
+  owner /{,var/}run/user/*/xauth_* r,
 
   # the unix socket to use to connect to the display
   /tmp/.X11-unix/* rw,
